.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,7 +6,17 @@ about: Create a report to help reproduce and fix the issue
 
 ## Issue Summary
 
-<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. -->
+<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. Screenshots can be especially helpful. -->
+
+<!-- Please help cull duplicate and confusing issues. Your answers here are not required unless specifically marked as such. -->
```

Comment:
Contributors don't need to know why we care about this. It just adds more lines to the template.